pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -168,6 +168,13 @@ func (m *kubeGenericRuntimeManager) configureContainerSwapResources(lcr *runtime
 		return
 	}
 	swapConfigurationHelper := newSwapConfigurationHelper(*m.machineInfo)
+	if m.memorySwapBehavior == kubelettypes.LimitedSwap {
```

Comment:
Just wondering if we are removing the support for `UnlimitedSwap` and `LimitedSwap` is the only type of `SwapBehavior` supported, then does it even need to be explicitly handled like this?

Reply:
Well we may have new options and we have `LimitedSwap` and `NoSwap`. Since `memorySwapBehavior` assumes "" or `NoSwap` I'm not sure how we can get away with not setting these. We should probably handle `LimitedSwap` and `NoSwap`. Users may either set to `NoSwap` or remove the option so we should probably support both.

Reply:
Makes sense, thanks.